duckduckgo/Android
# Improved and cleaned up the string resources, especially Russian translation

Task/Issue URL:  #559 

**Description**:
- Removed non-translatable strings from all translations except default string resources.
- Changed Play Store to the correct official name: Google Play.
- Slightly corrected Russian translation.
- Slightly cleaned up the code in the resources.

**Steps to test this PR**:
1. Check the changes in the source code.
3. Run the app.
4. Check the correct placement of the changed labels.


---
Regards,
Ilya. )
